float64 mx
float64 my
float64 eyex0
float64 eyey0
float64 eyex1
float64 eyey1
float64 eyex2
float64 eyey2
